modules/apps/object/object-rest-test/src/testFunctional/macros/UserAccountAPI.macro

```diff
@@ -223,8 +223,7 @@ definition {
 			givenName = ${givenName});
 
 		static var staticResponse = ${response};
-
-		static var staticUserAccountId = JSONUtil.getWithJSONPath(${staticResponse}, "$.id");
+		static var staticUserAccountId = JSONUtil.getWithJSONPath(${response}, "$.id");
```

Comment:
Does updating to JSONPathUtil.getIdValue(response = ${response}) works here?

Reply:
I will try, as I had some problems with static values and that's why omitted this 🤔 Checking now!

Reply:
I have tried to use JSONPathUtil.getIdValue(response = ${response}); here, and It can work well from my side.

Reply:
for some reason I get a message that the parameter is not set, I tried with response and with staticResponse, but still nothing. maybe it's because of where the macro file is in a relation of the testcase...

Reply:
@Hijky after updating, is the test BreakRelationshipBetweenSystemAndCustonObjects#CanBreakManyToManyRelationship running correctly for you? It is failing for me, and I have all feature flags enabled.

Reply:
@magjed4289 I don't know why today I failed, I will continue to try what cause the error, but I found an unnecessary echo in the CanBreakManyToManyRelationship test case
![Screenshot from 2023-02-24 09-53-40](https://user-images.githubusercontent.com/89900202/221072783-6a578f35-d19f-4cb1-ba32-6f52a749d268.png)

Reply:
@magjed4289 I found a method that can pass the test if you want to use JSONPathUtil.getIdValue(response = ${response}) you need to set these in the macro setUpGlobalUserAccountId

1. var userAccountId = JSONPathUtil.getIdValue(response = ${response});
2. static var staticUserAccountId = ${userAccountId};
3. return ${staticUserAccountId};

![Screenshot from 2023-02-24 11-06-51](https://user-images.githubusercontent.com/89900202/221082320-bc032f5d-9bf1-4d86-be06-894285fd8105.png)

Reply:
yup, this works, thanks for taking a look at it! 💟 I will update the PR :)